app/components/footer_component.html.erb

```diff
@@ -51,7 +51,7 @@
         <%= image_pack_tag 'static/dfelogo-white.svg', alt: "Department for Education", size: "124x73" %>
       </div>
       <div class="site-footer-bottom__links-container">
-        <a target="_blank" rel="noopener" href="https://forms.office.com/e/zVfV4SZeVn">Feedback</a>
+        <a target="_blank" rel="noopener" href="/feedback">Feedback</a>
```

Comment:
Feedback form should open in the same tab now that it is part of the site and not a separate office form @StackedPancakez